tools/commentMonitor/main.go

```diff
@@ -155,10 +155,22 @@ func (c *commentMonitorConfig) webhookExtract(w http.ResponseWriter, r *http.Req
 			return
 		}
 
+		// Strip whitespace.
+		cmClient.extractCommand()
```

Comment:
It is my personal preference, but I would think this would be more readable like

```suggestion
		c.ghClient.commentBody = extractCommand(c.ghClient.commentBody)
```
the same with all other places